lib/model/rwfolder.go

```diff
@@ -1505,7 +1519,7 @@ func (p *rwFolder) moveForConflict(name string) error {
 		err = nil
 	}
 	if p.maxConflicts > -1 {
-		matches, gerr := osutil.Glob(withoutExt + ".sync-conflict-????????-??????" + ext)
+		matches, gerr := osutil.Glob(withoutExt + "????????-??????" + ext)
```

Comment:
This should probably still match all conflicts for a given file, not just all conflicts caused by a given peer?